tools/GenerateHeaderForJson.ps1

```diff
@@ -2,34 +2,26 @@
 # the contents of that json files as a constexpr string_view in the header.
 
 param (
-    [parameter(Mandatory=$true, Position=0)]
+    [parameter(Mandatory = $true)]
     [string]$JsonFile,
 
-    [parameter(Mandatory=$true, Position=1)]
+    [parameter(Mandatory = $true)]
     [string]$OutPath,
 
-    [parameter(Mandatory=$true, Position=2)]
+    [parameter(Mandatory = $true)]
     [string]$VariableName
 )
 
-# Load the xml files.
+$fullPath = Resolve-Path $JsonFile
 $jsonData = Get-Content $JsonFile
 
-Write-Output "// Copyright (c) Microsoft Corporation" | Out-File -FilePath $OutPath -Encoding ASCII
-Write-Output "// Licensed under the MIT license." | Out-File -FilePath $OutPath -Encoding ASCII -Append
-Write-Output "" | Out-File -FilePath $OutPath -Encoding ASCII -Append
-Write-Output "// THIS IS AN AUTO-GENERATED FILE" | Out-File -FilePath $OutPath -Encoding ASCII -Append
-Write-Output "// Generated from " | Out-File -FilePath $OutPath -Encoding ASCII -Append -NoNewline
-$fullPath = Resolve-Path -Path $JsonFile
-Write-Output $fullPath.Path | Out-File -FilePath $OutPath -Encoding ASCII -Append
-Write-Output "constexpr std::string_view $($VariableName){ " | Out-File -FilePath $OutPath -Encoding ASCII -Append
-
-# Write each line escaped on its own, as it's own literal. This file is _very
-# big_, so big that it cannot fit in a single string literal :O The compiler is,
-# however, smart enough to just concatenate all these literals into one big
-# string.
-$jsonData | foreach {
-    Write-Output "R`"($_`n)`"" | Out-File -FilePath $OutPath -Encoding ASCII -Append
-}
-Write-Output "};" | Out-File -FilePath $OutPath -Encoding ASCII -Append
-
+@(
+    "// Copyright (c) Microsoft Corporation",
+    "// Licensed under the MIT license.",
+    "",
+    "// THIS IS AN AUTO-GENERATED FILE",
+    "// Generated from $($fullPath.Path)",
+    "constexpr std::string_view $($VariableName){",
+    ($jsonData | ForEach-Object { "R`"#($_`n)#`"" }),
+    "};"
+) | Out-File -FilePath $OutPath -Encoding utf8

```

Comment:
I had to add support for `"(` and `)"` escaping (now it uses `R"#(...)#"` instead), and I thought I could clean this file up a bit. 👍